test/testnet/localnet/config/deploy.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: "{{ lookup('env', 'TESTNET') }}"
+  become: yes
+  vars:
+    # Where, on the remote hosts, should we put the Tendermint home folder?
+    tmhome: "/home/tendermint/.tendermint"
```

Comment:
Ok, this is where things got hairy the last time too. As soon as we start implementing options that noone really cares about but that define how the service runs, the setup will get complicated and users will just drop it.

The point of the TNT setup was that it is _opinionated_. You don't get to choose where the config is, because it's already set up for you. Don't focus on the infrastructure because in 99% of the developer cases, you don't care about it. You care about the application (and to be honest I don't want to submit this to the tendermint repo because we might get away from EC2 instances before this becomes really useful for the team) not about where or how it's set up. (Within reason.)

I also want to minimize ansible playbooks - or better: make them as simple as possible. Look at the `networks/remote/ansible` folder in the SDK and the tendermint repos. Huge, complex, robust ansible scripts (I wrote them) and they are effectively useless. You need to be an ansible expert to understand what they are doing and why. You need to have the inside knowledge of how ansible folder structure affects ansible running to be able to figure out how exactly it is run.

If I could, I would simply stop using ansible for config management for these testnets. This will eventually happen, we're just not there yet.

Keep It Simple. Simple?